package.json

```diff
@@ -20,10 +20,6 @@
   ],
   "sideEffects": false,
   "scripts": {
-    "generate-component-install": "cd component-generator && npm install",
-    "consumer-app-install": "cd example && npm install",
-    "doc-site-install": "cd www && npm install",
-    "install-all": "npm install && npm run generate-component-install && npm run consumer-app-install && npm run doc-site-install",
     "build": "make build",
     "build-docs": "cd ./www && \"$npm_execpath\" install && \"$npm_execpath\" run build",
     "build-docs:with-theme": "cd ./www && \"$npm_execpath\" install && \"$npm_execpath\" run build:with-theme",
```

Comment:
I think we should remove this command, it is a legacy script and it's not used anymore, we can also remove `build:with-theme` script from the `www` package.

Reply:
Done